Library/Formula/libtorrent-rasterbar.rb

```diff
@@ -2,8 +2,8 @@
 
 class LibtorrentRasterbar < Formula
   homepage "http://sourceforge.net/projects/libtorrent/"
-  url "https://downloads.sourceforge.net/project/libtorrent/libtorrent/libtorrent-rasterbar-1.0.3.tar.gz"
-  sha1 "ccdd8bdba178b300921b15b18dfe8c0705f7eb07"
+  url "https://downloads.sourceforge.net/project/libtorrent/libtorrent/libtorrent-rasterbar-1.0.4.tar.gz"
+  sha1 "a2029f10e7ae9f8f1885c6cd308b737482c1d737"
```

Comment:
If you're feeling particularly kind, you could bump this to ` sha256 `. Don't worry if you don't want to though, It's not a huge thing or mandatory or such.